frappe/app.py

```diff
@@ -395,6 +396,9 @@ def handle_exception(e):
 		# don't fail silently for non-json response errors
 		print(frappe.get_traceback())
 
+	# Upload exception to Sentry
+	frappe.utils.sentry.capture_exception(e)
```

Comment:
For now lets do this for `frappe.log_error`. Tracking all exception is a lot of notice.